src/components/Header/Header.styled.js

```diff
@@ -25,10 +29,21 @@ export const HeaderSectionLeft = styled(HeaderSection)`
 
 export const HeaderSectionCenter = styled(HeaderSection)`
     justify-content: center;
+    padding: 0 16px;
+
+    @media(max-width: 768px){
+        width: 100%;
+        flex-shrink: .4;
+        /* flex-grow: 2; */
+    }
 `;
 
 export const HeaderSectionRight = styled(HeaderSection)`
     justify-content: flex-end;
+    padding-left: 16px;
+    @media(max-width: 768px){
+        display: none;
+    }
```

Comment:
alternativamente puedes crear una clase que contenga esta regla y agregársela a todos los elementos que quieras que se oculten en pantallas pequeñas para que no tengas que estar escribiendo esta regla una y otra vez en diferentes componentes, por ejemplo:
```
@media(max-width: 768px){
  .hidden-mobile {
    display: none;
  }
}
```

Reply:
Nice suggestion! Thank you! Implemented it in commit a49d16c